react/components/Slider.tsx

```diff
@@ -58,26 +60,36 @@ const Slider: FC<Props> = ({
       onTouchMove={e => (usePagination ? onTouchMove(e) : null)}
       aria-roledescription="carousel"
       aria-label={label}
-      style={{ WebkitOverflowScrolling: !usePagination ? 'touch' : undefined }}
-      className={`w-100 flex items-center relative ${
-        usePagination ? 'overflow-hidden' : 'overflow-x-scroll'
-      } ${handles.sliderLayoutContainer}`}
-      ref={containerRef}
+      style={{
+        WebkitOverflowScrolling: !usePagination ? 'touch' : undefined,
```

Comment:
Can you change the 'touch' to an enum?

Reply:
It already kinda is an enum hahaha this `WebkitOverflowScrolling` CSS property only supports two values: `'auto'` and `'touch'`, and TypeScript knows that

Reply:
https://developer.mozilla.org/pt-BR/docs/Web/CSS/-webkit-overflow-scrolling

Reply:
> Can you change the 'touch' to an enum?

I think is fine to use a string here, since `WebkitOverflowScrolling` is typed already and it has a set of values that it can take, you can't pass any string to it

Reply:
Oh ok!